src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/TaskResourceDao.java

```diff
@@ -85,13 +87,16 @@ private List<Selection<?>> getSelections(SearchTaskRequest searchTaskRequest, Ro
     }
 
     public List<TaskResource> getTaskResources(SearchTaskRequest searchTaskRequest,
-                                               List<TaskResourceSummary> taskResourcesSummary) {
+                                               List<Object[]> taskResourcesSummary) {
         SelectTaskResourceQueryBuilder selectQueryBuilder = new SelectTaskResourceQueryBuilder(entityManager, true);
         CriteriaBuilder builder = selectQueryBuilder.builder;
         Root<TaskResource> root = selectQueryBuilder.root;
 
         List<String> taskIds = taskResourcesSummary.stream()
-            .map(TaskResourceSummary::getTaskId)
+            .map(o -> stream(o).findFirst())
```

Comment:
This is all right. If someone change the possition on the taskId fields in select clause he will have painful few days to find out where the things go wrong :)